packages/storefront-lwc/src/modules/commerce/productAvailability/productAvailability.scss

```diff
@@ -1,3 +1,15 @@
-.availability {
-  margin-top: 1.071em;
+.pdp .availability {
```

Comment:
We should use bootstrap classes unless UX is adamant on a very specific value.

Reply:
okay, good point, I am researching on how...

Reply:
I actually can not use the bootstrap classes here because I need to have two different style on PDP and basket page.